roles/ipsec/tasks/main.yaml

```diff
@@ -0,0 +1,145 @@
+- name: install systemd-containers on ubunt
+  apt:
+      name: systemd-container
+      state: installed
+  when: ansible_os_family == "Debian"
+# nspawn binary path
+- name: set systemd nspawn binary path on ubuntu
```

Comment:
Use the vars dir for this.  Look at roles/kubernetes/preinstall/vars as an example.

There should be one file for each of: fedora, debian, redhat, centos.